pom.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.github.camel-tooling</groupId>
+	<artifactId>camel-common-tooling</artifactId>
```

Comment:
can we name it camel-tooling-common instead?

Reply:
it is to match the repository name.
Worth changing the repository name then?